frontend/src/routes/Infrastructure/Clusters/DiscoveredClusters/DiscoveryConfig/DiscoveryConfig.tsx

```diff
@@ -81,6 +90,20 @@ export default function DiscoveryConfigPage() {
     >
       <AcmPageContent id="discoveryConfigPageContent">
         <PageSection variant="light">
+          <Alert
+            variant="info"
+            isInline
+            title="Import your ROSA clusters automatically"
+            style={{ marginBottom: '1em' }}
+            className="pf-v5-u-mb-md"
+            actionLinks={[
+              returnLink(t('Policy example'), DOC_LINKS.ROSA_DISCOVERY_POLICY_EXAMPLE),
+              returnLink(t('Learn more'), DOC_LINKS.DISCOVERED_CLUSTERS),
+            ]}
```

Comment:
I don't like having to resort to the custom CSS for this. Looking at the [examples](https://v4-archive.patternfly.org/v4/components/alert#alert-variations), the spacing works properly if you use the `AlertActionLink` component. I looked into that component, and it is just a `Button` with `variant: link` and `isInline` props... so you can actually use a Button and it works. If you want to create a new component for external links, it could look something like the following.

```
actionLinks={[
              <AlertActionLink
                variant="link"
                isInline
                key="policy-example"
                component="a"
                href={DOC_LINKS.ROSA_DISCOVERY_POLICY_EXAMPLE}
                target="_blank"
                rel="noreferrer"
                icon={<ExternalLinkAltIcon />}
                iconPosition="right"
              >
                {t('Policy example')}
              </AlertActionLink>,
              <Button
                variant="link"
                isInline
                key="learn-more"
                component="a"
                href={DOC_LINKS.DISCOVERED_CLUSTERS}
                target="_blank"
                rel="noreferrer"
                icon={<ExternalLinkAltIcon />}
                iconPosition="right"
              >
                {t('Learn more')}
              </Button>,
            ]}
```